### ISSUE TITLE
arguments for enum with default_value are lowercase for introspection

### ISSUE THREAD

**Issue by Gazler** [absinthe-graphql/absinthe#412]
Consider the following schema:

```elixir
defmodule SomeSchema do
  use Absinthe.Schema.Notation

  enum :my_enum do
    value :foo
    value :bar
    value :baz
  end

    field :my_thing, :string do
      arg :my_arg, :my_enum, default_value: :foo
      resolve fn _, _, _ ->
        "return value"
      end
    end
end
```

When introspecting, this will return "FOO", "BAR" and "BAZ" as the enum values. However the default value for "my_arg" will be "foo". This is not a valid value for the argument, and will cause breakages in certain clients (the ruby graphql client in my case.)

Changing:

```elixir
      arg :my_arg, :my_enum, default_value: :foo
```

To:

```elixir
      arg :my_arg, :my_enum, default_value: :FOO
```

Fixes the issue.

**Comment by benwilson512** [comment]
It also shows the wrong value when simply looking up the enum type in GraphiQL.

**Comment by fsabe** [comment]
Hello. I'm learning Absinthe and this issue seemed simple enough to tackle. Do you mind taking a look at my solution?

**Comment by bruce** [comment]
I believe this is now resolved due to @fsabe's contribution in #413.
